packages/fxa-settings/src/models/Account.ts

```diff
@@ -559,17 +559,24 @@ export class Account implements AccountData {
     service?: string,
     redirectTo?: string
   ): Promise<PasswordForgotSendCodePayload> {
-    let serviceName;
+    let options: {
+      service?: string;
+      resume?: string;
+      redirectTo?: string;
+    } = {
+      resume: 'e30=', // base64 json for {}
```

Comment:
Okay, I'll add a note. I do wonder if this token is generally useful. It seems like it can be used to in a lot of scenarios where you need bounce some data between email links. Maybe this is irrelevant with the changes specified in FXA-6670.

Reply:
I believe it'll be irrelevant once we no longer have links. I know @vbudhram and I discussed it months back and we came to this conclusion anyway, which is why I'd pushed back on some of your resume token logic I think during the relier port (though, I was also under the impression we were going to be working on FXA-6670 pretty immediately😅).